blog.md

```diff
@@ -55,59 +64,93 @@ docker run --rm -ti -v /var/lib/dvol:/var/lib/dvol \\
     -v /var/run/docker.sock:/var/run/docker.sock \\
     clusterhq/dvol dvol \$@
 EOF
-# Install it
+```
+#### 3. Install it
+```
 sudo mv dvol /usr/local/bin/dvol
 sudo chmod +x /usr/local/bin/dvol
-# Try it out (should show no volumes)
-dvol list
 ```
-
-## How do I use voluminous?
-
-Voluminous volumes, like Docker containers, are global to your development machine.
-
-You manage Voluminous with a command line tool called `dvol`:
-
+#### 4. Have A Nice Day.
 ```
-$ dvol --help
+dvol --help
 Usage: dvol [options]
+Options:
+  -p, --pool=    The name of the directory to use
+      --version  Display Twisted version and exit.
+      --help     Display this help and exit.
 Commands:
-    list    List all volumes
-    init    Create a volume and its default master branch
-    commit  Create a commit on a branch
-    log     List commits on a branch
-    reset   Reset a branch to a given commit, throwing away more recent data
-$ dvol init frob_mysql
+    list          List all voluminous volumes
+    ls            Same as 'list'
+    init          Create a volume and its default master branch, then switch to
+                  it
+    switch        Switch active volume for commands below (commit, log etc)
+    rm            Destroy a voluminous volume
+    commit        Create a commit on the active volume and branch
+    log           List commits on the active volume and branch
+    reset         Reset active branch to a commit, destroying later unreferenced
+                  commits
+    branch        List or delete branches for active volume
+    checkout      Check out or create branches on the active volume
 ```
 
-You can start Docker containers with Voluminous volumes:
+## How do I use it?
 
+- *dvol CLI* : primary user interface for dvol, init, commit, reset, branch, etc.
+- *dvol Docker*: Volume Plugin* quickly connect your containers to your dvol volumes. `--volume-driver=dvol`
+- *dvol volumes*: like Docker containers, are global to your development machine.
+
+#### 1. init a new volume
+$ `dvol init frob_mysql`
```

Comment:
this suggests somthing with mysql, yet below we're running busybox. Does dvol working with all databases flocker/docker supports? If so, lets mention this somewhere